### DISCUSSION TITLE
How to contribute to Orchard Core Documentation

### DISCUSSION THREAD

**Discussion by agriffard** [OrchardCMS/OrchardCore#6883] [General]
If you want to edit or add some documentation, you can contribute by creating a PR.

This article explains how to contribute to the docs and even more, like running it locally to test your changes before submitting them:
https://docs.orchardcore.net/en/dev/docs/topics/docs-contributions/

You just need to know the Markdown syntax.
There are multiple editors that you can use like Visual Studio Code.

Documentation is an essential feature, an everlasting process and it is everyone's job to improve it.
Writing a documentation to expalin a concept that seems missing can be a good way to learn.
It is important to be able to find easily the information you are looking for.
It must be a part of each development.

Any help is appreciated.

**Comment by Piedone** [comment]
Also, if you're a newcomer, please especially consider contributing to the docs! Your experiences are especially valuable since you may notice or emphasize things that are trivial for more experienced Orchard users and thus they wouldn't document it but are still important.
